smdebug_rulesconfig/rule_config_jsons/ruleConfigs.json

```diff
@@ -203,5 +203,13 @@
         }
       }
     ]
+  },
+  "StalledTrainingRule" : {
```

Comment:
QQ: We don't need to provide what are parameters for this rule?

Reply:
> QQ: We don't need to provide what are parameters for this rule?

@Vikas-kum 
No. We do not need to  provide them. The Rule config package will vend out the default  configuration to run the rule.   In case of StalledTrainingRule, I noticed that 'threshold' is a tunable parameter. It already has a default value. Customers can still provide a different threshold for this rule by overriding the configuration vended by this pacakge.  This is explained here https://sagemaker.readthedocs.io/en/stable/amazon_sagemaker_debugger.html#pre-defined-debugger-hook-configuration-for-built-in-rules